web/src/settings_config.ts

```diff
@@ -77,11 +77,10 @@ export const user_list_style_values = {
         code: 2,
         description: $t({defaultMessage: "Show status text"}),
     },
-    // The `with_avatar` design in still in discussion.
-    // with_avatar: {
-    //     code: 3,
-    //     description: $t({defaultMessage: "Show status text and avatar"}),
-    // },
+    with_avatar: {
+        code: 3,
+        description: $t({defaultMessage: "Show avatar"}),
```

Comment:
I opted for this shorter string because the overflow didn't look great. Not sure how it looks with other languages, but at least now it's a comparable length to the other options.

Reply:
Probably worth posting the two options in the design thread for discussion, but I expect this may be fine.

Reply:
https://chat.zulip.org/#narrow/channel/101-design/topic/userlist.20style.20settings